### DISCUSSION TITLE
Calibrating 180 ohm tank sensor

### DISCUSSION THREAD

**Discussion by Ranur** [hatlabs/discussions#114] [General]
When connecting my tank sensor, I get a value in N2K, but when the sensor is higher than about 4/5 up, the value disappear from my plotter. The highest value I get before it disappear is 90.4% full. What do I have to do to get readings from the entire range of the sensor? I have measured it's value, and it is in fact 180 ohm.  Where do I start trouble shooting?
Also - what do I need to add to the sample code to get the tank sensor in signalk as well as n2k?
Best regards,
Runar

**Comment by mairas** [comment]
Speculating: HALMET (I'm assuming we're talking about it here) might read the tank value as a bit over 180 ohms when full, and if the result is beyond the interpolation range, it results an invalid value. To get around this, add one more set of values to the interpolation table:

    1000 1.0

This would result in any value between 180 ohm and 1kohm to be interpreted as a full tank (fill ratio of 1.0).

**Reply by Ranur** [reply]
Ok! I'll try that. I tried changing to 190 but that didn't cut it. So just to be clear 180 AND 1000 as 1?
How about tanks in signal K? I guess I'm currently missing a path. Anything else? My temp gauges show up in signalk
